### DISCUSSION TITLE
Chatbot multiple outputs

### DISCUSSION THREAD

**Discussion by jb11** [BrainJS/brain.js#824] [Q&A]
Is there a (best) way to train with input/output strings when there are multiple outputs for each input? I just got set up, so I am still making my way through the documentation, but I was testing with a simple input of "How are you?" and multiple possible outputs. But after I train with LSTM, the output is always the same for the input. Thanks

Edit: I actually found a video that answers my question. I figured it would be somewhat intent-based, which was confirmed.

**Comment by designerjason** [comment]
What is the video link...?

**Comment by goodwin74** [comment]
> Is there a (best) way to train with input/output strings when there are multiple outputs for each input? I just got set up, so I am still making my way through the documentation, but I was testing with a simple input of "How are you?" and multiple possible outputs. But after I train with LSTM, the output is always the same for the input. Thanks
> 
> Edit: I actually found a video that answers my question. I figured it would be somewhat intent-based, which was confirmed.

How did you eventually train the chat bot?
